frontend/src/components/common/ProfileSidebar.vue

```diff
@@ -22,6 +24,12 @@ const logout = () => {
     loading.value = true;
 };
 
+const showTermsBtn = () => {
```

Comment:
This name is confusing, this is an event, from the name it looks like to show the "button".